### PR TITLE
Update french (fr) translation

### PR DESCRIPTION
**Pull request by BoFFire**
Updating french (fr) translation via Weblate. 100% translated.

### CODE REVIEW

**Comment by mwarning** [comment]
Nice. If I have known you were that quick, then I should have waited with the release.

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -79,16 +79,16 @@
...
-    <string name="contact_name_empty">Contact name is empty.</string>
-    <string name="contact_name_exists">A contact with that name exists already.</string>
-    <string name="contact_remove">Remove contact?</string>
+    <string name="enter_contact_name">Entrez le nom du contact.</string>
+    <string name="wrong_password">Mot de passe non valide</string>
+    <string name="contact_name_conflict">Un contact ayant le même nom existe déjà.</string>
+    <string name="contact_key_conflict">Un contact ayant la même clé existe déjà.</string>
+    <string name="contact_name_empty">Le nom du contact est vide.</string>
+    <string name="contact_name_exists">Un contact ayant ce nom existe déjà.</string>
+    <string name="contact_remove">Supprimer le contact \?</string>
```
Comment (mwarning) [inline app/src/main/res/values-fr/strings.xml]: Is it normal in french to put a slash in front of the question mark?

**Comment by BoFFire** [comment]
@mwarning I'm using my own Weblate as a translation platform. If it is added, it mean that there's a reason somewhere related to the xml syntax. In my original strings, I have no slash :)

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (BoFFire) [reply app/src/main/res/values-fr/strings.xml]: @mwarning I'm using my own Weblate as a translation platform. If it is added, it mean that there's a reason somewhere related to the xml syntax. In my original strings, I have no slash :)

**Pull request merged**